rating-service/src/test/java/io/simakkoi9/ratingservice/e2e/steps/RatingSteps.java

```diff
@@ -0,0 +1,235 @@
+package io.simakkoi9.ratingservice.e2e.steps;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import io.simakkoi9.ratingservice.e2e.config.TestState;
+import io.simakkoi9.ratingservice.model.dto.rest.request.DriverRatingUpdateRequest;
+import io.simakkoi9.ratingservice.model.dto.rest.request.PassengerRatingUpdateRequest;
+import io.simakkoi9.ratingservice.model.entity.Rating;
+import io.simakkoi9.ratingservice.repository.RateRepository;
+import io.simakkoi9.ratingservice.repository.RatingRepository;
+import io.simakkoi9.ratingservice.util.TestDataUtil;
+import jakarta.inject.Inject;
+import jakarta.transaction.Transactional;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.response.Response;
+import org.apache.http.HttpStatus;
+
+public class RatingSteps {
+
+    @Inject
+    TestState testState;
+
+    @Inject
+    RatingRepository ratingRepository;
+
+    @Inject
+    RateRepository rateRepository;
+
+    private Response response;
+
+    @Given("the database is clean")
+    public void theDatabaseIsClean() {
+        testState.cleanDatabase();
+    }
+
+    @Given("a completed ride with ID {string}")
+    public void aCompletedRideWithId(String rideId) {
+        testState.setCurrentRideId(rideId);
+    }
+
+    @When("I create a rating for the ride")
+    public void iCreateARatingForTheRide() {
+        response = given()
+            .contentType("application/json")
```

Comment:
Здесь и далее:
```suggestion
            .contentType(ContentType.JSON)
```
Следует заменить строку константой, как это было в `RatingControllerIT`.